applications/dotwise.md

```diff
@@ -0,0 +1,191 @@
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 2
+
+## Project Overview :page_facing_up:
+
+This is a response for the [Analytics Website/Data Platform](https://github.com/w3f/Grants-Program/blob/master/docs/RFPs/Open/analysis-website-and-data-platform.md) RFP.
+
+### Overview
+
+![logo](https://github.com/tolgayayci/Grants-Program/assets/40897846/b8d20b5f-dd67-436a-8066-04aebf674a97)
+
+Dotwise aims to develop a user-friendly platform for on-chain analysis in the Polkadot & Kusama ecosystems, inspired by the utility and design of Dune Analytics which is popular in the Ethereum community. The platform will facilitate easy exploration and querying of blockchain data via backend services such as Subquery, using GraphQL. The key feature of this platform will be its capability to let users create, share, and customize dashboards for data visualization and interpretation.
+
+### Project Details
+
+- Mockups/designs of any UI components
+
+  **Home Page**
+  
+  ![homepage](https://github.com/tolgayayci/Grants-Program/assets/40897846/b2100e98-827d-4f49-a6c7-78f64c148ed1)
+
+  **Queries:** Query is core feature of the platform. Users will be able to create on-chain queries with the Subquery API via Dotwise Core API.
+
+  ![query](https://github.com/tolgayayci/Grants-Program/assets/40897846/59c8acc9-8460-4743-bd12-8daf67d78551)
+
+  **Visualizations:** Users will be able to create visualizations with the queries and visualize the data with the built-in visualization tools.
+
+  **Dashboards:** Dashboards are the main showcase page of the platform. Dashboards has some widgets to display such as queries, visualizations, tables, images, and text. Drag and drop feature will be available for the widgets.
+  
+  ![dashboard](https://github.com/tolgayayci/Grants-Program/assets/40897846/527d4f0f-c0c0-4044-96f3-a9a8f820d2ed)
+
+  **P.S:** This mockups designed for inspiration. The final design can be different from this mockups.
+
+- Data models / API specifications of the core functionality
+
+  **Core API:** Dotwise Core API will be developed with [Fast API](https://fastapi.tiangolo.com/) and it will be used for core functionalities. It will be used by the frontend and backend services.
+
+  **SubQuery API:** SubQuery API will be used for querying the data from the Substrate-based networks.
+
+- An overview of the technology stack to be used
+
+  ![techstack](https://github.com/tolgayayci/Grants-Program/assets/40897846/21ed9c58-1950-4e8a-bf28-9e484a1eaf4b)
+
+  **Subquery:** Subquery is a tool for building custom data indexes for Polkadot, Kusama, and other Substrate-based networks. It is a great tool for building custom data indexes. It provides a great developer experience with many built-in features such as GraphQL support, TypeScript support, and Substrate support. It also has a great community and ecosystem with many plugins and tools.
```

Comment:
Can you elaborate how this data index looks like? Do you just tell SubQuery to index everything and then any query I can think of is available? Or do complex queries require a customisation of this index?

Reply:
I will use the subquery to index the customized data and then save it to my database, and for this I need to determine its scope and proceed in this way. Indexing and querying everything is difficult and not a very possible task for this project. Instead, I aim to create detailed indexes under core infrastructure such as nft details, account analysis, token analysis and so more.

Users can write their own queries on these indexed data, or they can create queries by simply selecting the attributes and related properties without writing a query with the module I will create.